gulp/tasks/coverage.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    var gulp = require('gulp'),
+        coveralls = require('gulp-coveralls');
+
+    gulp.task('coverage', function() {
+        // Only run on Travis
+        if (process.argv.indexOf('--ci') !== -1) {
```

Comment:
What happens if this is run locally?  If it fails, a check to see if process exists might be helpful--maybe even with a warning message.